### PR TITLE
Release updates

### PR DESCRIPTION
**Pull request by daniel-lau-mf**
Update charts to support use of idol-library 0.14.0.
See #66 for new features.
Updates image version to 24.3.

### CODE REVIEW

File: helm/single-content/README.md [hunk 1]
```diff
@@ -9,41 +9,32 @@ a single IDOL Content instance. Intended as a much lighter-weight alternative to
 the `distributed-idol` chart, for use in testing charts that expect one or both
 of these endpoints to exist in the cluster.
 
-> Full documentation for Content available from https://www.microfocus.com/documentation/idol/IDOL_24.1/Content_24.1_Documentation/Help/
+> Full documentation for Content available from https://www.microfocus.com/documentation/idol/IDOL_24.3/Content_24.3_Documentation/Help/
```
[the remark this answers is not in the document]
Comment (joe-wood-opentext) [reply helm/single-content/README.md:12]: ditto for answerserver/eductionserver/qms

File: helm/idol-eductionserver/values.schema.json [hunk 2]
```diff
@@ -196,15 +207,45 @@
...
                 }
             },
-            "required": [
+            "required":[
                 "idolImage",
                 "ingress",
                 "name",
                 "aciPort",
                 "servicePort",
-                "grammarPackage"
```
Comment (joe-wood-opentext) [inline helm/idol-eductionserver/values.schema.json:207]: Why was `grammarPackage` made non-required?

**Review by joe-wood-opentext — APPROVED** [review]

File: helm/idol-eductionserver/values.schema.json (hunk 2 above)
Comment (daniel-lau-mf) [reply helm/idol-eductionserver/values.schema.json:207]: `grammarPackage` is defined and required within the `eductionserver` object [helm/idol-eductionserver/values.schema.json#L264-L287](https://github.com/opentext-idol/idol-containers-toolkit/blob/5c13d71c10c087fffdb9c603ee477dd345d4a986/helm/idol-eductionserver/values.schema.json#L264-L287) - the occurrence here was just in the wrong place.

File: helm/single-content/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (daniel-lau-mf) [reply helm/single-content/README.md]: [045af3d](https://github.com/opentext-idol/idol-containers-toolkit/pull/69/commits/045af3d9361ffa45e65b33af6f95e33ce24c3a45)

**Comment by daniel-lau-mf** [comment]
@lewis-mackay-mf - could you sanity check that I've resolved the issues that @joe-wood-opentext raised? If happy then I can create the release packages and merge this.

**Review by lewis-mackay-mf — APPROVED** [review]

**Pull request merged**